src/Aquifer.Public.API/Endpoints/Bibles/List/Response.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Text.Json.Serialization;
+using Aquifer.Common.Utilities;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.List;
+
+public record Response
+{
+    public required int Id { get; set; }
+    public required string Name { get; set; }
+    public required string Abbreviation { get; set; }
+    public required int LanguageId { get; set; }
+    public required bool IsLanguageDefault { get; set; }
+    public required bool HasGreekAlignment { get; set; }
+    public required bool HasAudio { get; set; }
```

Comment:
I'm a big fan of `bool` property names beginning with `should`, `is`, `has`, etc. because it uses naming convention to convey that the variable is a binary option.  For example, I'd expect a property named `Audio` to contain audio information, not to be a flag telling me if the Bible has audio data.  However, this naming philosophy is not consistent with other responses.

I can change this to be consistent if desired but I wanted to raise the discussion.

Reply:
+1

Reply:
100% agree.